Add showreel 2023 submissions open blogpost (#755)
godotengine/godot-website: collections/_article/submissions-open-godot-2023-showreel.md

---
title: "Submissions open for Godot 2023 showreel!"
excerpt: "Showcase your Godot creations in 2023! We're inviting submissions of short videos featuring your published or in-progress Godot games or tools. Don't miss this chance to be a part of it!"
categories: ["news"]
author: Emi
image: /storage/blog/covers/godot-showreel-2023.webp
date: 2023-11-20 00:00:00
---

> Showcase your Godot Engine 2023 creations! We invite you to submit videos featuring your published or in-progress games or tools so we can make the Godot 2023 Showreeel.

Reviewer: ```suggestion
Showcase your Godot Engine 2023 creations! We invite you to submit videos featuring your published or in-progress games or tools so we can make the Godot 2023 Showreel.
```
eee :)

> Showcase your Godot Engine 2023 creations! We're invite you to submit videos featuring your published or in-progress games or tools so we can make the Godot 2023 Showreeel.

Reviewer 2: ```suggestion
Showcase your Godot Engine 2023 creations! We invite you to submit videos featuring your published or in-progress games or tools so we can make the Godot 2023 Showreeel.
```

Author: thanks!

> The showreels are an informative and celebratory showcase, to highlight the wide array of projects developed using the Godot Engine. If you want to participate and share your Godot-made game, app or tool submit your entry starting now.

Reviewer: ```suggestion
The showreels are an informative and celebratory showcase, to highlight the wide array of projects developed using the Godot Engine. If you want to participate and share your Godot-made game, app or tool, submit your entry by December 17th 2023.
```
"starting now" sounded a bit odd (sounds a bit like starting a race), so I initially changed to just "now". But then that sounded too immediate, so I changed it to as above!

Could alternatively say "submit your entry in the next four weeks".

Reviewer 3: I think the point is to announce that the submissions are now open. So it can be "starting today" if it's better to your ear.

Reviewer: Yes, "starting today" does sound better than "starting now". Or you could just say directly "submissions are now open".

I'm coming round more to "submit your entry in the next four weeks" though, as it implies that submissions are open, and also gives a little hint on how long they've got, before going into the full details with UTC timezone etc. below.

Author: Replaced with starting today 👍

Reviewer: OK thanks :). Don't forget the comma after "tool".

> ## How to Submit:

Reviewer: ```suggestion
## How to submit
```

You can submit your entry in our [Godot Showreel website](https://showreel.godotengine.org/). You can use your Godot account or any other account (GitHub, Discord, etc.) to log in. Before submitting, please read the [submission guidelines](https://showreel.godotengine.org/about).

- ✅ **[Submit your entry](https://showreel.godotengine.org/)**
- 📋 **[Submission guidelines](https://showreel.godotengine.org/about)**
- 📅 **Submissions close on December 17th 2023 at 23:59 UTC**

> ## Need Inspiration?

Reviewer: ```suggestion
## Need inspiration?
```
